rslib/src/browser_table.rs

```diff
@@ -575,6 +575,10 @@ impl RowContext {
                         Color::Marked
                     } else if self.cards[0].queue == CardQueue::Suspended {
                         Color::Suspended
+                    } else if self.cards[0].queue == CardQueue::UserBuried
+                        || self.cards[0].queue == CardQueue::SchedBuried
+                    {
```

Comment:
This section could be simplified a little with:

```rust
                _ => {
                    if self.note.is_marked() {
                        Color::Marked
                    } else {
                        match self.cards[0].queue {
                            CardQueue::Suspended => Color::Suspended,
                            CardQueue::UserBuried | CardQueue::SchedBuried => Color::Buried,
                            _ => Color::Default,
                        }
                    }
                }
```

Reply:
I like ctrl-shift-j for "toggle bury". It's similar to ctrl-j for "toggle suspend". Commit suggested changes.